src/sections/stats/sections/POL/PieWrapper/PieWrapper.tsx

```diff
@@ -0,0 +1,83 @@
+import { PieSkeleton } from "sections/stats/components/PieChart/components/Skeleton/Skeleton"
+import { ChartSwitchMobile } from "sections/stats/components/ChartSwitchMobile/ChartSwitchMobile"
+import { useMedia } from "react-use"
+import { theme } from "theme"
+import { useMemo, useState } from "react"
+import { BN_0 } from "utils/constants"
+import { ChartWrapper } from "sections/stats/components/ChartsWrapper/ChartsWrapper"
+import { SContainerVertical } from "../StatsPOL.styled"
+import { PieTotalValue } from "../../overview/components/PieTotalValue/PieTotalValue"
+import { TUseOmnipoolAssetDetailsData } from "../../../StatsPage.utils"
+import { PieChart } from "../../../components/PieChart/PieChart"
+
+type PieWrapperProps = {
+  data: TUseOmnipoolAssetDetailsData
+  isLoading: boolean
+}
+
+export const PieWrapper = ({ data, isLoading }: PieWrapperProps) => {
+  const isDesktop = useMedia(theme.viewport.gte.sm)
+  const [activeSection, setActiveSection] = useState<"overview" | "chart">(
+    "overview",
+  )
+
+  const totalPol = useMemo(
+    () =>
+      Object.values(data ?? {}).reduce(
```

Comment:
data is an array, I don't think you have to do that. Just call reduce fn on data array